### PR TITLE
Add Configuration Page

### LINKED ISSUE #3: feat: add scheduler page
### Feature request:

add scheduler page

### PR DESCRIPTION
**Pull request by lucy-cl**
## Description
Add Configuration Page

## Related Issue
Fixes #3

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [x] Code compiles correctly.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .umirc.ts [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
-        { exact: true, path: '/', component: '@/pages/index' },
-        { exact: true, path: '/config', component: '@/pages/config' },
+        { exact: true, path: '/', component: '@/pages/scheduler' },
+        { exact: true, path: '/scheduler', component: '@/pages/scheduler' },
+        { exact: true, path: '/cdn', component: '@/pages/cdn' },
       ],
     },
   ],
+  locale: {
+    default: 'en-US', //默认语言 en-US
```
Comment (gaius-qi) [inline .umirc.ts:23]: Use english

File: package.json [hunk 2]
```diff
@@ -20,10 +20,11 @@
     ]
   },
   "dependencies": {
-    "@alicloudfe/components": "^1.1.102",
-    "@alicloudfe/components-console-layout": "1.1.0",
+    "@ant-design/icons": "^4.6.3",
```
Comment (gaius-qi) [inline package.json:23]: Remove `@ant-design/icons` package.

File: src/global.css [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
 #root {
-  width: 100%;
   height: 100%;
 }
-.avatar {
-  width: 24px;
-  height: 24px;
-  border-radius: 50%;
-  vertical-align: middle;
+.ant-layout {
```
Comment (gaius-qi) [inline src/global.css:4]: Remove

File: src/layouts/index.jsx [hunk 4]
```diff
@@ -1,51 +1,154 @@
...
+    const temp = location.pathname.split('/')[1];
+    setKey(temp.length ? [temp] : ['scheduler']);
+    const jwt2 = getCookie('jwt2');
+    const userInfo = decode(jwt2, 'jwt');
+    const { id } = userInfo;
+    getUserById(id);
+  }, [location.pathname]);
+
+  const getCookie = (name) => {
+    const dict = document.cookie
```
Comment (gaius-qi) [inline src/layouts/index.jsx:39]: Use `js-cookie` package.

https://github.com/js-cookie/js-cookie

File: .umirc.ts [hunk 5]
```diff
@@ -1,20 +1,36 @@
...
+    baseNavigator: true, // 为true时，用navigator.language的值作为默认语言
+    antd: true, // 是否启用antd的<LocaleProvider />
+  },
   fastRefresh: {},
+  proxy: {
+    '/api/v1': {
+      target: 'http://11.122.75.66:8080', // dep env
```
Comment (gaius-qi) [inline .umirc.ts:30]: Remove IP.

File: src/layouts/index.jsx [hunk 6]
```diff
@@ -1,51 +1,154 @@
...
+  history,
+  match,
+}) {
+  const [key, setKey] = useState(['scheduler']);
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
+    const temp = location.pathname.split('/')[1];
+    setKey(temp.length ? [temp] : ['scheduler']);
+    const jwt2 = getCookie('jwt2');
```
Comment (gaius-qi) [inline src/layouts/index.jsx:32]: Why use `jwt2`.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: src/layouts/index.jsx [hunk 7]
```diff
@@ -1,51 +1,154 @@
...
+import {
+  Layout,
+  Menu,
+  Breadcrumb,
+  Divider,
+  Avatar,
+  Dropdown,
+  Button,
+} from 'antd';
+import { SettingOutlined } from '@ant-design/icons';
```
Comment (gaius-qi) [inline src/layouts/index.jsx:11]: Use `import { Icon } from 'antd';`

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: src/layouts/index.jsx [hunk 8]
```diff
@@ -1,51 +1,154 @@
...
+  location,
+  route,
+  history,
+  match,
+}) {
+  const [key, setKey] = useState(['scheduler']);
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
+    const temp = location.pathname.split('/')[1];
```
Comment (gaius-qi) [inline src/layouts/index.jsx:30]: Use SafeGet.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (lucy-cl) [reply package.json:23]: antd 4.x update icon to @ant-design/icons

File: src/layouts/index.jsx (hunk 7 above)
Comment (lucy-cl) [reply src/layouts/index.jsx:11]: antd 4.x update icon to @ant-design/icons

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by jim3ma — APPROVED** [review]

**Pull request merged**